doc/rules/control_structure/fully_multiline.rst

```diff
@@ -0,0 +1,94 @@
+========================
+Rule ``fully_multiline``
+========================
+
+Multi-line arrays, arguments list, parameters list and control structures,
+``switch`` cases and ``match`` expressions should have one element by line.
+
+Configuration
+-------------
+
+``elements``
+~~~~~~~~~~~~
+
+Which expression must have one element by line (PHP >= 8.0 for ``parameters``
+and ``match``).
+
+Allowed values: a subset of ``['arguments', 'arrays', 'case', 'control_structures', 'match', 'parameters']``
+
+Default value: ``['arrays']``
```

Comment:
Not sure about the default value:
- Since this rule is kinda similar/related to the TrailingCommaInMultilineFixer, I copied the default value
- But if we're drive by the PSR12, the default value might be `control_structure` only.

Reply:
As this is a new rule, what about enabling everything by default?